### DISCUSSION TITLE
The datagrid scrollbar cannot scroll down properly

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#14688] [Q&A] [answered]
Version:11.0.7
PS：It is in a popup control.
https://github.com/AvaloniaUI/Avalonia/assets/76192669/d18a35fd-9fb8-40d9-b8ff-a1de389b7135

**Comment by maxkatz6** [comment, marked as answer]
You should limit size of the datagrid, so it can properly virtualize its content.
